code/retrieval/retriever.py

```diff
@@ -344,16 +619,17 @@ async def search(self, query: str, site: Union[str, List[str]],
         Returns:
             List of search results
         """
```

Comment:
Jen start here - this is where you left off

Reply:
i assume this is a note to yourself

Reply:
yes it is sorry